vue-frontend/src/components/contributions/UnitTest.vue

```diff
@@ -0,0 +1,225 @@
+<template>
+  <div class="tw-my-[1rem] lg:tw-mt-[5rem] tw-mx-auto tw-text-center">
+    <span
+      class="tw-container tw-text-center tw-font-inter-bold tw-text-black-core/[0.8] tw-text-[1.875rem] lg:tw-text-[3.4375rem] tw-leading-[2.4375rem] lg:tw-leading-[5.15625rem]"
+      >WHY and How to Write Unit Tests?</span
+    >
+    <div
+      class="tw-block lg:tw-hidden swiper-content tw-mt-16 tw-h-full tw-pl-[5%]"
+    >
+      <swiper
+        :slidesPerView="1.1"
+        :centeredSlides="false"
+        :autoplay="{
+          delay: 4000,
+          disableOnInteraction: true,
+        }"
+        :speed="3000"
+        :loop="true"
+        :loopedSlides="50"
+        :spaceBetween="10"
+        :pagination="pagination"
+        class="swiper-container"
+      >
+        <swiper-slide
+          v-for="(test, index) in tests"
+          :key="index"
+          class="tw-cursor-pointer"
+        >
+          <div class="tw-cursor-pointer" @click="openBlog(test.url)">
+            <img
+              :src="test.image[0]"
+              :srcset="`${test.image[0]} 400w,${test.image[1]} 800w`"
+              class="tw-w-fit tw-h-fit tw-object-contain tw-border-b-[0.5px] tw-rounded-t-lg tw-border-black-core/[0.87] tw-drop-shadow-md cover-image"
+              alt="UnitTestImage"
+            />
+            <div
+              class="tw-mb-[2rem] tw-p-[20px] tw-bg-white tw-rounded-b-lg tw-drop-shadow-md content"
+            >
+              <span class="tw-font-inter-medium tw-text-black-core/[0.87]">{{
+                test.title
+              }}</span>
+            </div>
+          </div>
+        </swiper-slide>
+      </swiper>
+    </div>
+    <!-- ************************Desktop View****************************** -->
+    <div
+      class="tw-hidden lg:tw-block tw-mt-[3rem] xl:tw-mt-16 tw-bg-gradient-to-t tw-from-[#FF835B]/[0.1] tw-via-[#F2709C]/[0.1] tw-to-[#FFFFFF]"
+    >
+      <div class="xll:tw-container swiper-content tw--mt-6">
+        <swiper
+          :slidesPerView="3"
+          :centeredSlides="true"
+          :autoplay="{
+            delay: 2000,
+            disableOnInteraction: false,
+          }"
+          :loop="true"
+          :loopedSlides="50"
+          :navigation="true"
+          :pagination="pagination"
+          :breakpoints="{
+            '768': {
+              slidesPerView: 1,
+            },
+          }"
+          class="swiper-container"
+        >
+          <swiper-slide
+            ><div
+              class="tw-mt-[1rem] tw-border-t-[0.5px] tw-border-black-core/[0.4]"
+            ></div>
+            <img
+              :src="dot"
```

Comment:
Dot animation is behind then images.

[Screencast from 20-06-23 11:56:59 AM IST.webm](https://github.com/canopas/canopas-website/assets/69897605/0d56f0b8-0a92-48c4-a58d-a812142557c5)

Let's not use separate swipers for dot animation. Attach them with images